sdk/storage/azure-storage-files-shares/src/share_file_client.cpp

```diff
@@ -939,7 +941,9 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
     auto uploadPageFunc = [&](int64_t offset, int64_t length, int64_t chunkId, int64_t numChunks) {
       (void)chunkId;
       (void)numChunks;
-      Azure::Core::IO::MemoryBodyStream contentStream(buffer + offset, length);
+      // TODO: Investigate changing lambda parameters to be size_t, unless they need to be int64_t
+      // for some reason.
```

Comment:
Can you open an issue and assign it to me?

Reply:
Sure.

Reply:
https://github.com/Azure/azure-sdk-for-cpp/issues/2313